tests/languages/tap/plan_feature.test

```diff
@@ -0,0 +1,11 @@
+1..10
```

Comment:
The `plan` pattern in the language definition seems to also include an optional directive (e.g. `# foobar`) after the numbers part. If this is what is intended, we should add a test for it here.